### PR TITLE
upgrade terraform version to 1.4.5 and refactored kitchen test

### PR DESCRIPTION
**Pull request by babuharidass**

### CODE REVIEW

File: ci/variables/env [hunk 1]
```diff
@@ -0,0 +1 @@
+DEFAULT_KITCHEN_TERRAFORM_TAG=2.0.1
```
Comment (alv-lop) [inline ci/variables/env:1]: Same comment as https://github.com/dwp/terraform-aws-kong-gateway/pull/103#discussion_r1172247312

Comment (babuharidass) [reply ci/variables/env]: removed the file

File: Makefile [hunk 2]
```diff
@@ -19,16 +19,21 @@ git-hooks: ## Set up hooks in .githooks
...
+	@ci/scripts/run-kitchen.sh --module concourse --action verify --args ${test_suites}
 
 .PHONY: destroy
 destroy: ## Build default scenario with Kitchen Terraform
-	docker run --rm -e AWS_PROFILE=default -v $(shell pwd):/usr/action -v ~/.aws:/root/.aws -v /etc/ssl/certs/:/usr/local/share/ca-certificates/ quay.io/dwp/kitchen-terraform:0.14.7 "destroy $(test_suites)"
+	@ci/scripts/run-kitchen.sh --module concourse --action destroy --args ${test_suites}
+
+.PHONY: debug
+debug: ## Debug hybrid-mode scenario with Kitchen Terraform
+	@ci/scripts/run-kitchen.sh --module concourse --action debug --args ${test_suites}
```
[the remark this answers is not in the document]
Comment (babuharidass) [reply Makefile]: removed the module

File: ci/scripts/run-kitchen.sh [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+      --volume $(pwd):/usr/action \
+      --volume ~/.aws:/root/.aws \
+      --user root \
+      --workdir /usr/action/ \
+      ${IMAGE} "${ACTION}  ${ARGS}"
+  fi
+
+else 
+  echo "The following arguments are required: \`--module\` and \`--action\`."
+fi
```
[the remark this answers is not in the document]
Comment (babuharidass) [reply ci/scripts/run-kitchen.sh]: removed the module as suggested

**Review by dan-hill2802 — APPROVED** [review]

**Pull request merged**